bot/exts/moderation/watchchannels/talentpool.py

```diff
@@ -246,6 +246,8 @@ def _nomination_to_string(self, nomination_object: dict) -> str:
         log.debug(active)
         log.debug(type(nomination_object["inserted_at"]))
```

Comment:
I know this isn't a part of this PR, but I think these debug logs should be removed. They seem very out of place.
```suggestion
```